examples/albert/requirements.txt

```diff
@@ -1,7 +1,7 @@
 transformers~=4.6
 datasets~=1.5
 torch_optimizer==0.1.0
-wandb==0.10.26
+wandb==0.17.1
```

Comment:
This does not seem to be relevant to the Pydantic upgrade :)

Reply:
feel free to merge this pr for the albert example https://github.com/learning-at-home/hivemind/pull/611